src/app/shared/utils.ts

```diff
@@ -0,0 +1,18 @@
+import * as constants from '../constants';
+
+export function kelvinToCelsius(temperature: number): number {
+    return temperature - constants.CELSIUS_KELVIN_DIFFERENCE;
+}
+
+export function kelvinToFahrenheit(temperature: number): number {
+    return 9 / 5 * temperature - constants.KELVIN_FAHRENHEIT_DIFFERENCE;
```

Comment:
could you please move 9 and 5 to constants? or 9/5
it's not easy to get what 9/5 means